_includes/footer.html

```diff
@@ -170,24 +170,30 @@ <h2 class="h6">
                         </noscript>
                     </div>
                     <p class="footer_address">
-                        <span class="footer_address-word-break">
-                            Consumer Financial
+                        <span itemprop="name">
+                            <span class="footer_address-word-break">
+                                Consumer Financial
+                            </span>
+                            Protection Bureau<br>
+                        </span>
+                        <span itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+                            <span itemprop="streetAddress">1700 G Street, NW<br></span>
```

Comment:
@jimmynotjim, should I go ahead and update the macros with the schema properties?